AirCasting/Dashboard/ReorderingSessionHeader.swift

```diff
@@ -0,0 +1,94 @@
+// Created by Lunar on 28/01/2022.
+//
+
+import SwiftUI
+
+struct ReorderingSessionHeader: View {
+    var session: SessionEntity
+    
+    var body: some View {
+        sessionHeader
+    }
+}
+
+private extension ReorderingSessionHeader {
+    var sessionHeader: some View {
+        VStack(alignment: .leading, spacing: 3) {
+            HStack {
+                dateAndTime
+                    .foregroundColor(Color.aircastingTimeGray)
+                Spacer()
+                Image("draggable-icon")
+            }
+            nameLabel
+        }
+        .font(Fonts.regularHeading4)
+        .foregroundColor(.aircastingGray)
+    }
+
+    var dateAndTime: some View {
+        adaptTimeAndDate()
+    }
+    
+    var nameLabel: some View {
+        VStack(alignment: .leading, spacing: 3) {
+            HStack {
+                Text(session.name ?? "")
+                    .font(Fonts.regularHeading1)
+                Spacer()
+            }
+            sensorType
+                .font(Fonts.regularHeading4)
+        }
+        .foregroundColor(.darkBlue)
+    }
+    
+    var sensorType: some View {
```

Comment:
this very same code used in both dashboard and here. I think this particular case is a good one for extracting into a separate `View`:
```Swift
struct SensorTypeIndicator: View {
    var body: some View  { here this code from here }
}
```